node/src/components/consensus/highway_core/highway_testing.rs

```diff
@@ -244,7 +244,10 @@ impl HighwayValidator {
                                 // TODO: Don't send both messages to every peer. Add different
                                 // strategies.
                                 let mut wunit = swunit.wire_unit.clone();
-                                wunit.timestamp += 1.into();
+                                match wunit.value.as_mut() {
+                                    None => wunit.timestamp += 1.into(),
+                                    Some(v) => v.push(0),
+                                }
```

Comment:
I could swear we already did exactly that. :sweat_smile: (Maybe in a PR that was closed?)

Reply:
Yes, it was closed.